examples/adwords/v201708/optimization/get_keyword_bid_simulations.py

```diff
@@ -58,7 +58,7 @@ def main(client, ad_group_id, criterion_id):
   # Display results.
   if 'entries' in bid_landscapes:
     for bid_landscape in bid_landscapes['entries']:
-      if bid_landscape['BidLandscape.Type'] == 'CriterionBidLandscape':
+      if bid_landscape['DataEntry.Type'] == 'CriterionBidLandscape':
```

Comment:
On closer inspection, I don't think this check does anything useful. Entries returned from `getCriterionBidLandscape` are certainly going to be `CriterionBidLandscape`. It would probably be better to just remove the check entirely.